cobalt/build/get_build_id_test.py

```diff
@@ -87,8 +87,9 @@ def testGetBuildNumberFromCommitCountSunnyDay(self):
       self.make_commit()
     build_number = build_info.get_build_id_from_commit_count(
         cwd=self.test_dir.name)
-    self.assertEqual(build_number,
-                     num_commits + build_info.COMMIT_COUNT_BUILD_ID_OFFSET)
+    self.assertEqual(
```

Comment:
If we do keep it a string, is there some assert method we can use which will allow different types? i.e. do the cast automatically?

Reply:
a super preliminary search doesn't seem to bring up any automatic casting asserts, probably by design